src/apps/trainings/models/network.py

```diff
@@ -215,7 +215,7 @@ class Meta:
     )
 
     def __str__(self):
-        return f"{self.name} ({self.elo}±{2 * self.elo_uncertainty})"
+        return f"{self.name} ({self.elo}±{2 * self.elo_uncertainty:.1})"
```

Comment:
can you extract the ".1" to a variable like precision and replace 
`self.elo_uncertainty:.1` by `self.elo_uncertainty:.{precision}f`

that way we can reuse this precision for line 226 and 234

Reply:
Done. However, I feel it might be not easier to read.